api/src/main/java/ai/djl/nn/norm/GhostBatchNorm.java

```diff
@@ -0,0 +1,121 @@
+package ai.djl.nn.norm;
+
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.training.ParameterStore;
+import ai.djl.translate.Batchifier;
+import ai.djl.translate.StackBatchifier;
+import ai.djl.util.PairList;
+
+/**
+ * {@link GhostBatchNorm} is similar to {@link BatchNorm} except that it splits a batch into a
+ * smaller sub-batches aka <em>ghost batches</em>, and normalize them individually to have a mean of
+ * 0 and variance of 1 and finally concatenate them again to a single batch. Each of the
+ * mini-batches contains a virtualBatchSize samples.
+ */
+public class GhostBatchNorm extends BatchNorm {
+
+    private int virtualBatchSize;
+    private Batchifier batchifier;
+
+    GhostBatchNorm(Builder builder) {
+        super(new BatchNorm.Builder());
```

Comment:
```suggestion
        super(builder);
```

Otherwise, any setting you make for BatchNorm such as `optAxis()` or `optScale()` would not be set properly